### ISSUE TITLE
add groups in layer list in openlayers export

### ISSUE THREAD

**Issue by ordisabeba** [tomchadwin/qgis2web#977]
is there a solution to create groups that group layers?

**Comment by ordisabeba** [comment]
I found a solution by editing the layers.js file adding this example code:

```
var layersList = [
lyr_test_0,
new ol.layer.Group({
            title: 'Title',
            fold: 'open',
            layers: [
new ol.layer.Group({
            title: 'Subtitle',
            fold: 'open',
            layers: [
lyr_test_1,lyr_test_2,
]})
]})
];
```

**Comment by andreaordonselli** [comment]
I have just released version [3.19.0](https://github.com/qgis2web/qgis2web/releases/tag/v3.19.0), which fixes the issues discussed in this thread. Shortly, the update will be available within the QGIS plugin repository.

Thank you for using the plugin.

I would greatly appreciate it if you would consider donating a virtual coffee through the button on the homepage to support the development of qgis2web.

Best regards.
